content/source/docs/enterprise/before-installing/reference-architecture/vmware.md

```diff
@@ -100,6 +103,11 @@ providing an auto-recovery mechanism in the event of virtual machine or physical
 The Storage Layer is provided in the form of attached disk space configured with or benefiting from inherent resiliency
 provided by the NAS or SAN.
 
+-> **Important Note** The storage device/service used must be high-speed in both I/O and connectivity and highly reliable to meet performance requirements. A SAN device will generally meet these requirements. Many standard NAS and other device types, however, will not perform at the level required. So only use a NAS or other device type if it can  accommodate these requirements. The specific selection and configuration of the storage device is not covered in this document.
```

Comment:
```suggestion
-> **Note:** Terraform Enterprise's storage device or service must be highly reliable and high-speed in both I/O and connectivity to meet performance requirements. Device types in the supported list will usually meet these requirements, but many standard NAS and other device types will not perform at the level required. Only use a NAS or other device type not in the supported list if you are certain it can accommodate these requirements.

The specific selection and configuration of the storage device is not covered in this document.
```